lib/editor/components/ScheduleExceptionForm.js

```diff
@@ -122,12 +122,30 @@ export default class ScheduleExceptionForm extends Component<Props> {
     } = activeEntity
 
     // getRangesForDates requires a sorted array. We also sort the zeroes to the end of the array for display purposes.
+    let addDateDisabled = false // Disable adding a new date if we come across any current validation issues.
```

Comment:
Do we need an explicit variable for this? Since we're using it only in one place can't we just count the number of validation errors inside the `disabled` value?

Reply:
We do not, I'll update this

Reply:
Updated in https://github.com/ibi-group/datatools-ui/pull/843/commits/bcfe4911d3d622f0ebf397e12c47e6b12df3ea45